core/blockchain/blockchain_gateway.py

```diff
@@ -1,15 +1,9 @@
 import asyncio
-import base58
-import json
-import logging
-import os
-import requests
-import time
 
 import ipfsapi
 
-from core.configuration import ConfigurationManager
-from core.utils.event_types import MessageEventTypes
+from core.blockchain.blockchain_utils import *
+from core.utils.event_types import *
```

Comment:
I would highly recommend not using `import *`. The person working on this file does not immediately know what is being imported, which could mean that: 

1. One might accidentally rename an imported method
2. Given a class or method being used in this file, it is not obvious whether it comes from this file or some other file (since imported classes/methods aren't explicit in the import statement).